x-pack/test/functional/page_objects/canvas_page.ts

```diff
@@ -32,6 +32,14 @@ export function CanvasPageProvider({ getService, getPageObjects }: FtrProviderCo
       await testSubjects.findAll('canvasWorkpadPage > canvasWorkpadPageElementContent');
     },
 
+    async loadSavedWorkpad(workpadName: string) {
+      const elem = await testSubjects.find('canvasWorkpadLoaderWorkpad');
+      const text = await elem.getVisibleText();
+      expect(text).to.be(workpadName);
```

Comment:
Note: this function only works because it's used in a scenario where there is only 1 workpad in the list, and it is the one we are looking for.